NumberTheory/Fibonacci.cpp

```diff
@@ -3,6 +3,12 @@
     Find the N-th Fibonacci number, given the value of N.
     (The Fibonacci sequence is characterized by the fact that every number
     after the first two is the sum of the two preceding ones.)
+
+    Time complexity:
+    O(n), where n is the value for which Fibonacci number must be generated
```

Comment:
Use `N` instead of `n`.

And I think the following:
> where N is the term of the Fibonacci sequence to calculate

would better describe what `N` is. What do you think?